### DISCUSSION TITLE
How do we monitor guests ?

### DISCUSSION THREAD

**Discussion by selroc** [cloud-hypervisor/cloud-hypervisor#3455] [Q&A] [answered]
What software must we use to monitor guests?

**Comment by rbradford** [comment, marked as answer]
@selroc What are you interested in from the guests? We have some counters available (`ch-remote counters`)?

**Reply by selroc** [reply]
This should be sufficient to get an oversight of guest workload, and I can feed counter values to any reporting system.
Thank you.

**Reply by rbradford** [reply]
The HTTP endpoint is called `vm.counters` if you want to pull it directly from the API.

**Reply by selroc** [reply]
> The HTTP endpoint is called `vm.counters` if you want to pull it directly from the API.

I will set up a cron job that regularly feeds counter values to my reporting system. I do this with plain text output.
